### PR TITLE
[AIRFLOW-6205] The 'airflow db shell' works on mysql without port/passwd

### PR DESCRIPTION
**Pull request by potiuk**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-6205

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

The `airflow db shell' command now works also for mysql with no port/password

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### CODE REVIEW

**Comment by potiuk** [comment]
@mik-laj -> found it while cherry-picking to v1-10-test

**Comment by KKcorps** [comment]
Shouldn't we allow an empty value for all params? In the code, it's already allowed for Postgres.

**Review by mik-laj — APPROVED** [review]

File: airflow/cli/commands/db_command.py [hunk 1]
```diff
@@ -61,8 +61,8 @@ def shell(args):
                 [client]
                 host     = {url.host}
                 user     = {url.username}
```
Comment (mik-laj) [inline airflow/cli/commands/db_command.py:63]:
```suggestion
                user     = {url.username or ""}
```
Is Username  optionally?

**Comment by mik-laj** [comment]
@potiuk Any progress on it?

**Comment by potiuk** [comment]
Not really . Have another, more important stuff to work on. Care to take over?

**Pull request merged**